src/open_inwoner/openzaak/tests/test_migrations.py

```diff
@@ -66,3 +70,47 @@ def test_migration_0048_to_0051_multi_zgw_backend(self):
             expected,
             msg="Service config should have been moved to a new ZGWApiGroupConfig",
         )
+
+
+class TestMakeZaakTypeConfigCatalogusRequired(TestCase):
```

Comment:
We have a utility class for migration tests that sets + resets the constraint checks (among other things). You don't have to use it, but take a look: https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/utils/tests/test_migrations.py